README.md

```diff
@@ -1,2 +1,489 @@
-# lbaas-api-docs
-Equinix LBaaS API Documentation 
+> [!WARNING]  
+> This software is currently in a beta release. For this reason, we don't recommend this for immediate usage while running production workloads. Any feedback is greatly appreciated!
+
+# Equinix Load Balancers API Quickstart
+
+[![Stability: Beta](https://img.shields.io/badge/stability-beta-33bbff.svg)]()
+
+This document will describe how you can get started with Equinix Load Balancers via the API.
+
+## Authentication
+
+Before getting started, we'll gather the necessary components we'll need to be able to make our API calls. This will include the ID of the project that we wish to create loadbalancers in, an API token and then finally exchanging our API token for a JWT that will be used to make any requests.
+
+### Get Project ID
+
+#### Console URL
+If you're not already familiar with the project ID that you want to use, you can get this from the URL in your console UI:
+
+```bash
+https://console.equinix.com/projects/<project-uuid>
+```
+
+#### Metal CLI
+
+If you're already using the `metal` [CLI](https://github.com/equinix/metal-cli/tree/main), you can also get your project ID using the following:
```

Comment:
We could skip a few more steps with `metal` CLI already configured.

`eval $(metal env --export)` .. This will put the default (from console) METAL_PROJECT_ID and METAL_AUTH_TOKEN in the environment.